packaging/systemd/cloudstack-agent.default

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+JAVA_HOME=/usr/lib/jvm/jre
+if [ ! -f "$JAVA_HOME"/bin/java ] ; then
+    JAVA_HOME=$(readlink -f /usr/bin/java | sed 's:/bin/java::')
+fi
```

Comment:
Thanks @vincentbernat that's a nice idea. I've removed usage of JAVA_HOME, I checked and indeed all we want was path to java which I checked by default is at /usr/bin/java on most distros including the ones I tested - Ubuntu 12.04, 14.04, 16.04 and on CentOS 7(.2). Are we good to merge with the new change?